pk3DS.Core/Randomizers/PersonalRandomizer.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Reflection;
```

Comment:
Are you using anything in this namespace? I don't see anything, but I could have missed something.